### ISSUE TITLE
freeview surfaces not loading

### ISSUE THREAD

**Issue by jordandekraker** [khanlab/hippunfold#89]
I believe this stems from the transition from v0.5.3 to v0.5.4

When I try to load output surfaces to freeview, I get an 'invalid surface' error (the command line text outputs an error in  mrisReadGIFTIfile, but doesn't say anything more descriptive). I'm seeing this with freeview 2.0 and 3.0 (2.0 gives a different but equally unhelpful error message). I am still able to load the original unfolded template files though. When I compare the original template files with the hippunfold output files using nibabel's `print_summary()`, I noticed these differences:
hippunfold output:
```
DataArray 0:
Intent:  NIFTI_INTENT_POINTSET
DataType:  NIFTI_TYPE_FLOAT32
ArrayIndexingOrder:  RowMajorOrder
...
DataArray 1:
Intent:  NIFTI_INTENT_TRIANGLE
DataType:  NIFTI_TYPE_INT32
ArrayIndexingOrder:  RowMajorOrder
```
original:
```
DataArray 0:
Intent:  NIFTI_INTENT_TRIANGLE
DataType:  NIFTI_TYPE_INT32
ArrayIndexingOrder:  ColumnMajorOrder
...
DataArray 1:
Intent:  NIFTI_INTENT_POINTSET
DataType:  NIFTI_TYPE_FLOAT32
ArrayIndexingOrder:  ColumnMajorOrder
```

Will try to narrow down the issue further soon.

**Comment by akhanf** [comment]
@royhaast just slacked me about some NaN vertices causing issues -- maybe that is the case here too? 
Are you able to load in the unfold_template (flat) surfaces into Freeview? If so, then it might be the NaN's that are getting introduced later on..

**Comment by jordandekraker** [comment]
That sounds likely. I am indeed able to load the unfold_template (flat) surfaces. Also the error seems to occur somewhat inconsistently, and more commonly is freeview 3.0. Sometimes it works in one instance and then crashes with the same files in the next instance!

**Comment by jordandekraker** [comment]
I'll start a PR to interpolate nan vertices based on connected vertices (since we can't remove them)

**Comment by jordandekraker** [comment]
See https://github.com/khanlab/hippunfold/pull/90

**Comment by akhanf** [comment]
should be fixed in #90, @jordandekraker feel free to close this if so
